CMakeLists.txt

```diff
@@ -244,8 +244,8 @@ else()
 	# Once the next version of yaml-cpp is released (first version not requiring
```

Comment:
Can you remove these comments as they aren't relevant anymore?

Reply:
done.